contracts/libs/Types.sol

```diff
@@ -36,7 +36,7 @@ library Types {
     struct Commitment {
         bytes32 stateRoot;
         bytes32 accountRoot;
-        bytes32 txHashCommitment;
+        bytes txs;
```

Comment:
moving bytes txs below to signature could be better practice in order not to disturb properties with full evm sizes